File: tests/interfile/java/test_java_constructor_split/FieldUser.java
class FieldUser {
    public String name;

    public FieldUser() {
        this.name = "";
    }

    public String getProfile() {
        // ruleid: java_constructor_sqli
        String query = "SELECT * FROM users WHERE name = " + this.name;
        return query;
    }
}

File: tests/interfile/php/test_php_constructor_split/FieldUser.php
<?php

class FieldUser {
    public $name;

    public function __construct() {
        $this->name = "";
    }
    
    public function getProfile() {
        // ruleid: php_constructor_sqli
        $query = "SELECT * FROM users WHERE name = " . $this->name;
        return $query;
    }
}
